### PR TITLE
Kp/run

### PR DESCRIPTION
**Pull request by kprokofi**

### CODE REVIEW

**Comment by kprokofi** [comment]
Added AugMixGrey
Created different searching spaces for MNV3 and EffNet
Deleted hardcoded paths from configs

File: torchreid/utils/tools.py [hunk 1]
```diff
@@ -127,7 +127,7 @@ def read_image(path, grayscale=False):
             img = Image.open(path).convert('L' if grayscale else 'RGB')
             got_img = True
         except IOError:
-            print('IOError incurred when reading "{}". Will redo. Don\'t worry. Just chill.'.format(path))
+            print('IOError incurred when reading "{}".'.format(path))
```
Comment (sovrasov) [inline torchreid/utils/tools.py]: occurred

File: torchreid/optim/lr_finder.py [hunk 2]
```diff
@@ -56,7 +58,8 @@ def __init__(self,
         self.enable_sam = engine.enable_sam
         self.smooth_f = smooth_f
         self.engine_cfg = Dict(min_lr=min_lr, max_lr=max_lr, mode=mode, step=step)
-        self.samplers = {'grid_search': GridSampler(search_space={'lr': [0.005, 0.007, 0.01, 0.015, 0.02, 0.025, 0.03]}),
```
Comment (sovrasov) [inline torchreid/optim/lr_finder.py:59]: Why do you need to narrow the search space for EfficientNets?

Comment (kprokofi) [reply torchreid/optim/lr_finder.py:59]: I don't narrow the search space for EfficientNet. I use np.arrange(0.001, 0.011).  So, 10 numbers to search.

Comment (sovrasov) [reply torchreid/optim/lr_finder.py:59]: I meant narrow in terms of range. This silent replacement makes behavior of the class unpredictable to user, so I suggest avoiding such tricks.

File: configs/classification/EfficientNet_b0.yml [hunk 3]
```diff
@@ -8,7 +8,7 @@ lr_finder:
   smooth_f: 0.01
   epochs_warmup: 1
   path_to_savefig: 'lr_finder.jpg'
-  max_lr: 0.01
```
Comment (sovrasov) [inline configs/classification/EfficientNet_b0.yml]: Very strange replacement. Is the method so unstable?

Comment (kprokofi) [reply configs/classification/EfficientNet_b0.yml]: It is not related to stability. Just if we want to use step size = 0.002, 0.009 is the max number that we can reach. In this case, if we set 0.01, Optuna throws a warning.

Anyway, I set the step size equals to 0.001 and the upper bound to 0.01

File: torchreid/optim/lr_finder.py (hunk 2 above)
Comment (kprokofi) [reply torchreid/optim/lr_finder.py:59]: I fixed that. We will use the np.arange(min_lr, max_lr, step) and we can manage this behavior through the config file

**Review by sovrasov — APPROVED** [review]

**Pull request merged**